prime-router/src/main/kotlin/cli/FileUtilities.kt

```diff
@@ -85,13 +81,22 @@ class FileUtilities {
             val outputFile = if (outputFileName != null) {
                 File(outputFileName)
             } else {
+                // is this config HL7?
+                val hl7Config = report.destination?.translation as? Hl7Configuration?
+                // if it is, get the test processing mode
+                val processingMode = if (hl7Config?.useTestProcessingMode == false) {
+                    "P"
+                } else {
+                    "T"
+                }
                 val fileName = Report.formFilename(
```

Comment:
This code kind of repeats in both `FileUtilities` and `main`. Feels like we should refactor put in a common place like the `Hl7Configuration`?